.circleci/config.yml

```diff
@@ -175,6 +176,7 @@ workflows:
             parameters:
               arch: [ x64 ]
       - mac-build:
+          context: fiddle-release
```

Comment:
I think we'll eventually need this context in all build steps but approving since Mac is ready now

Reply:
Yea, once we move the remaining secrets. I'm contemplating making a context per-platform just to silo them a bit more and only providing them on more of a "need to know" basis.